api/src/interpreter/dto/paginate-interpreter-query.dto.ts

```diff
@@ -79,12 +79,20 @@ export class PaginateInterpreterQueryDto extends PaginationQueryDTO {
   @IsOptional()
   keywords?: string;
 
+  @ApiProperty({
+    description: 'Interpreter Active/Inactive indicator',
+    example: faker.random.boolean(),
+  })
+  @IsOptional()
+  active?: string;
```

Comment:
not type boolean?

Reply:
Raw query generator utility unable to generate query for boolean value (specially for case false), but works fine if I supply this as string.